editor/src/document/document_message.rs

```diff
@@ -99,6 +99,10 @@ pub enum DocumentMessage {
 		layer_path: Vec<LayerId>,
 		set_expanded: bool,
 	},
+	SetLayerName {
```

Comment:
I noticed a `RenameLayer` operation already exists. See if that can be used/reused/removed to avoid duplication.

Reply:
Thanks for noticing this! It never came up in my search. All the more reason for nice documentation to be a goal where we can generate a Rustdoc documentation website to preview all these things nicely :) Thanks for the code review.